test/components/octicon_component_test.rb

```diff
@@ -56,13 +56,13 @@ def test_renders_default_size_small
   end
 
   def test_renders_the_provided_size
-    render_inline(Primer::OcticonComponent.new(:star, size: :large))
+    render_inline(Primer::OcticonComponent.new(:star, size: :medium))
 
-    assert_selector("[height='64']")
+    assert_selector("[height='24']")
   end
 
   def test_renders_with_overridden_height_and_width_despite_given_a_size
```

Comment:
I'd be interested to hear if we still want to allow this override.

Reply:
I don't think we should, but this was added after the fact in https://github.com/primer/view_components/pull/100 

Searching github/github there's not too many places that don't use standard sizes. But this does bring up the fact that  a lot of places are using `height` and not `size` eg. `<%= render Primer::OcticonComponent.new(icon: "squirrel", height: 24) %>`

* https://github.com/github/github/blob/197c0ccba0f24c060163f423c39d205758bb720c/app/components/sponsors/dependencies_banner_component.html.erb#L5
* https://github.com/github/github/blob/197c0ccba0f24c060163f423c39d205758bb720c/app/components/sponsors/profile/corporate_sponsor_beta_discount_component.html.erb#L7
* https://github.com/github/github/blob/197c0ccba0f24c060163f423c39d205758bb720c/app/views/businesses/billing_settings/budgets/_budget_filters.html.erb#L11
* https://github.com/github/github/blob/197c0ccba0f24c060163f423c39d205758bb720c/app/views/layouts/bootstrap_instance.html.erb#L12

Reply:
After more searching, I think we should work up a longer term plan to deprecate the support of custom sizes. There's a lot of places in dotcom that use `height:` instead of `size:` and there's a few instances where a valid icon size 24 would break because they're using height: 32